### PR TITLE
WET-464 -> Topic theme with stacked header

### PR DESCRIPTION
**Pull request by SebastianBurke**

### CODE REVIEW

File: templates/topic/topic-stackedheader-en.html
Comment (duboisp) [inline templates/topic/topic-stackedheader-en.html:1]: This one need to become the default "topic-en.html"

And the current page "topic-en.html" need to be renamed.

**Review by duboisp — CHANGES_REQUESTED** [review]
And the following jekyll alert need to be inserted in the documentation page, at the top

```
{%- include /alert-softlaunch.html genre="m" component="topic template" version="2" -%}
```

```
{%- include /alert-softlaunch.html genre="m" component="gabarit de sujet" version="2" -%}
```

@Ricokola once the example is updated, please spin up a working example for our BA. Thanks

File: templates/topic/topic-en.html [hunk 1]
```diff
@@ -1,15 +1,16 @@
 ---
 title: "[Topic title]"
+sectionTitle: "[Section title]"
 layout: no-container
 language: "en"
-altLangPage: "topic-fr.html"
-dateModified: "2024-06-10"
+altLangPage: "topic-stackedheader-fr.html"
```
Comment (duboisp) [inline templates/topic/topic-en.html:6]:
```suggestion
altLangPage: "topic-fr.html"
```

File: templates/topic/topic-fr.html [hunk 2]
```diff
@@ -1,15 +1,16 @@
 ---
 title: "[Titre du sujet]"
+sectionTitle: "[Titre de la section]"
 layout: no-container
 language: "fr"
-altLangPage: "topic-en.html"
-dateModified: "2024-06-10"
+altLangPage: "topic-stackedheader-en.html"
```
Comment (duboisp) [inline templates/topic/topic-fr.html:6]:
```suggestion
altLangPage: "topic-en.html"
```

File: templates/topic/topic-nostackedheader-en.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+title: "[Topic title]"
+layout: no-container
+language: "en"
+altLangPage: "topic-fr.html"
```
Comment (duboisp) [inline templates/topic/topic-nostackedheader-en.html:5]:
```suggestion
altLangPage: "topic-nostackedheader-fr.html"
```

File: templates/topic/topic-nostackedheader-fr.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: "[Titre du sujet]"
+layout: no-container
+language: "fr"
+altLangPage: "topic-en.html"
```
Comment (duboisp) [inline templates/topic/topic-nostackedheader-fr.html:5]:
```suggestion
altLangPage: "topic-nostackedheader-en.html"
```

File: templates/topic/topic-nostackedheader-fr.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+dateModified: "2024-06-10"
+---
+
+<div class="container">
+	<div class="row">
+		<div class="col-md-6">
+			<h1 property="name" id="wb-cont">{{ page.title }}</h1>
```
Comment (duboisp) [inline templates/topic/topic-nostackedheader-fr.html:12]: Use the include like you did for "topic-en,html"

File: templates/topic/topic-nostackedheader-en.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+dateModified: "2024-06-10"
+---
+
+<div class="container">
+	<div class="row">
+		<div class="col-md-6">
+			<h1 property="name" id="wb-cont">{{ page.title }}</h1>
```
Comment (duboisp) [inline templates/topic/topic-nostackedheader-en.html:12]: Use the include like you did for "topic-en,html"

**Review by duboisp — CHANGES_REQUESTED** [review]

File: _data/templates.json [hunk 7]
```diff
@@ -1355,6 +1365,39 @@
...
+			],
+			"implementation": [
+				"_:implement_topic",
+				"_:implement_topic_aem",
+				"_:implement_topic_1_to_2"
+			],
+			"history": [
+				{
+					"en": "August 2024 - Version 1 of the stacked header variant.",
+					"fr": "août 2024 - Version 1 de la variante d'en-tête empilé."
```
Comment (Ricokola) [inline _data/templates.json:1397]: For consistency, I suggest replacing the word "empilé" by "superposé"

File: _data/templates.json [hunk 8]
```diff
@@ -1463,9 +1506,51 @@
 					"Veuillez suivre les instructions d'implémentation sous l'onglet «&nbsp;Standard&nbsp;»\"."
 				]
 			}
+		},
+		{
+			"@id": "_:implement_topic_stackedheader",
+			"iteration": "_:iteration_topic_4",
+			"name": {
+				"en": "Topic page with stacked header",
+				"fr": "Page de sujet avec en-tête superposée"
```
Comment (Ricokola) [inline _data/templates.json:1515]: "en-tête" is masculine so "superposée" should not have an extra "e" at the end.

File: _data/templates.json [hunk 9]
```diff
@@ -1463,9 +1506,51 @@
...
+				"en": [
+					{
+						"href": "topic-en.html",
+						"text": "Topic page with stacked header"
+					}
+				],
+				"fr": [
+					{
+						"href": "topic-fr.html",
+						"text": "Page de sujet avec en-tête superposée"
```
Comment (Ricokola) [inline _data/templates.json:1549]: "en-tête" is masculine so "superposée" should not have an extra "e" at the end.

File: templates/topic/index.json-ld [hunk 10]
```diff
@@ -192,9 +235,51 @@
 					"Veuillez suivre les instructions d'implémentation sous l'onglet «&nbsp;Standard&nbsp;»\"."
 				]
 			}
+		},
+		{
+			"@id": "_:implement_topic_stackedheader",
+			"iteration": "_:iteration_topic_4",
+			"name": {
+				"en": "Topic page with stacked header",
+				"fr": "Page de sujet avec en-tête superposée"
```
Comment (Ricokola) [inline templates/topic/index.json-ld:244]: "en-tête" is masculine so "superposée" should not have an extra "e" at the end.

File: templates/topic/index.json-ld [hunk 11]
```diff
@@ -192,9 +235,51 @@
...
+				"en": [
+					{
+						"href": "topic-en.html",
+						"text": "Topic page with stacked header"
+					}
+				],
+				"fr": [
+					{
+						"href": "topic-fr.html",
+						"text": "Page de sujet avec en-tête superposée"
```
Comment (Ricokola) [inline templates/topic/index.json-ld:278]: "en-tête" is masculine so "superposée" should not have an extra "e" at the end.

**Review by Ricokola — CHANGES_REQUESTED** [review]
Great work! Just few minor content changes.

File: _data/templates.json [hunk 12]
```diff
@@ -1301,6 +1301,16 @@
 				"language": "fr",
 				"path": "topic-fr.html"
 			},
+			{
+				"title": "[Topic title without stacked header]",
+				"language": "en",
+				"path": "topic-nostackedheader-en.html"
+			},
+			{
+				"title": "[Titre du sujet sans en-tête superposée]",
```
Comment (Ricokola) [inline _data/templates.json:1310]: Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"

File: _data/templates.json [hunk 13]
```diff
@@ -1355,6 +1365,39 @@
 	},
 	"a11yGuidance": "No accessibility guidance.",
 	"variations": [
+		{
+			"name": {
+				"en": "Topic page with stacked header",
+				"fr": "Page de sujet avec en-tête superposée"
```
Comment (Ricokola) [inline _data/templates.json:1371]: Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"

File: _data/templates.json [hunk 14]
```diff
@@ -1355,6 +1365,39 @@
...
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/mandatory-templates/theme-topic.html",
+				"fr": "https://conception.canada.ca/modeles-obligatoire/theme-sujet.html"
+			},
+			"iteration": "_:iteration_topic_4",
+			"example": [
+				{
+					"en": { "href": "topic-en.html", "text": "Topic page with stacked header" },
+					"fr": { "href": "topic-fr.html", "text": "Page de sujet avec en-tête superposée" }
```
Comment (Ricokola) [inline _data/templates.json:1386]: Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"

File: _data/templates.json (hunk 8 above)
Comment (Ricokola) [inline _data/templates.json:1515]: Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"

File: _data/templates.json (hunk 9 above)
Comment (Ricokola) [inline _data/templates.json:1549]: Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"

**Review by Ricokola — CHANGES_REQUESTED** [review]
I've identified few remaining content changes. Can you please confirm when they will be adjusted.

**Review by duboisp — CHANGES_REQUESTED** [review]
The "template.json" can be simply re-generated during the release.

**Review by Garneauma — CHANGES_REQUESTED** [review]
Please move the variation with stacked header below the the variation without since it is no longer the default one.

File: templates/topic/index.json-ld [hunk 15]
```diff
@@ -84,6 +94,39 @@
...
+			"status": "stable",
+			"description": {
+				"en": "Use the stacked main page title to display the page's section title as well as the page title.",
+				"fr": "Utilisez le titre de la page principale superposé pour afficher le titre de la section de la page ainsi que le titre de la page."
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/mandatory-templates/theme-topic.html",
+				"fr": "https://conception.canada.ca/modeles-obligatoire/theme-sujet.html"
+			},
+			"iteration": "_:iteration_topic_4",
```
Comment (Garneauma) [inline templates/topic/index.json-ld:111]:
```suggestion
			"iteration": "_:iteration_topic_3",
```

File: templates/topic/index.json-ld [hunk 16]
```diff
@@ -192,6 +237,30 @@
 					"Veuillez suivre les instructions d'implémentation sous l'onglet «&nbsp;Standard&nbsp;»\"."
 				]
 			}
+		},
+		{
+			"@id": "_:implement_topic_stackedheader",
+			"iteration": "_:iteration_topic_4",
```
Comment (Garneauma) [inline templates/topic/index.json-ld:243]:
```suggestion
			"iteration": "_:iteration_topic_3",
```

**Review by Garneauma — CHANGES_REQUESTED** [review]

File: templates/topic/index.json-ld [hunk 17]
```diff
@@ -124,6 +134,39 @@
 					"fr": "Selon le gabarit en date de Janvier 2018 - Implémentation de la version 1 du gabarit de page."
 				}
 			]
+		},
+		{
+			"name": {
+				"en": "Topic page with stacked header",
+				"fr": "Page de sujet avec en-tête superposé"
```
Comment (Garneauma) [inline templates/topic/index.json-ld:141]:
```suggestion
				"en": "Standard",
				"fr": "Standard"
```

**Review by Garneauma — CHANGES_REQUESTED** [review]
I think that's the only thing left. Unless @duboisp sees something I don't.

File: templates/topic/index.json-ld [hunk 18]
```diff
@@ -192,6 +237,30 @@
 					"Veuillez suivre les instructions d'implémentation sous l'onglet «&nbsp;Standard&nbsp;»\"."
 				]
 			}
+		},
+		{
+			"@id": "_:implement_topic_stackedheader",
+			"iteration": "_:iteration_topic_3",
+			"name": {
+				"en": "Topic page with stacked header",
+				"fr": "Page de sujet avec en-tête superposé"
```
Comment (Garneauma) [inline templates/topic/index.json-ld:246]:
```suggestion
				"en": "Standard",
				"fr": "Standard"
```

**Review by Garneauma — CHANGES_REQUESTED** [review]

File: templates/topic/index.json-ld (hunk 18 above)
Comment (Garneauma) [reply templates/topic/index.json-ld]: @SebastianBurke Capital "S" please.

File: _includes/settings.liquid
Comment (duboisp) [inline _includes/settings.liquid:1]: Ce fichier doit être exclu

File: sites/main-page-title/_base.scss
Comment (duboisp) [inline sites/main-page-title/_base.scss:1]: Ce PR ne doit pas contenir les changements d'introduction du Stacked header donc ce fichier ne devrait pas être modifier

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — CHANGES_REQUESTED** [review]
@Garneauma can you check the commit history and fix it?

**Review by duboisp — COMMENTED** [review]
Other than re-organizing the commit history, the propose change look good.

**Comment by Garneauma** [comment]
WET-464 was approved by PP and DTO as of now.

**Pull request merged**